static/build.html

```diff
@@ -1032,7 +1032,14 @@ <h3><a href="#prebuilt-apps">Prebuilt apps</a></h3>
                     <p>The official releases of our App Store, Auditor, Camera and PdfViewer apps are
                     bundled as apks into external/ repositories. The no-code AppCompatConfig and
                     GmsCompatConfig apps are done the same way. These are built and signed with the
-                    standard <code>gradle</code> Android plugin build system.</p>
+                    standard <code>gradle</code> Android plugin build system. You can also build those apps from the source code by following these steps:
+                    </p>
+                    <ul>
+                        <li>Install Gradle, OpenJDK 21 for developers (which should be called <code>java-21-openjdk-devel</code> on Linux), Android Studio, and npm.</li>
```

Comment:
This is only the name of the package on Debian distributions and isn't the name on Arch Linux or elsewhere. On Arch Linux, the package name is `jdk21-openjdk` by comparison with `jre21-openjdk` as the subset without all the development tools. Debian has a standard devel split they use but it's not clear exactly what they did in this case. Arch Linux normally doesn't split headers, etc. into a separate package but the JDK is a special case.

Reply:
Good point, thanks. Would this be ok?
```suggestion
                        <li>Install Gradle, Android Studio, npm, and OpenJDK 21 (<code>java-21-openjdk-devel</code> package on Debian distributions, <code>jdk21-openjdk</code> on Arch Linux distributions...).</li>
```